deluge/core/core.py

```diff
@@ -822,6 +859,11 @@ def get_listen_port(self) -> int:
         """Returns the active listen port"""
         return self.session.listen_port()
 
+    @export
+    def get_ssl_listen_port(self) -> int:
+        """Returns the active SSL listen port"""
+        return self.session.ssl_listen_port()
```

Comment:
This is a tricky one since we are both adding a new RPC method and relying on a new libtorrent exposed method...

Sometimes we check for specific libtorrent version but it likely is easier in this case to catch the AttributeError and log a warning. For the clients we could either re-raise the exception with a nice message or return an invalid port number, not sure which would be better long-term

```suggestion
        try:
            return self.session.ssl_listen_port()
       except AttributeError:
            log.warning("libtorrent version >=2.x required to get active SSL listen port")
            return -1
```